package.json

```diff
@@ -43,7 +43,7 @@
     "meow": "^3.3.0",
     "mime-types": "^2.1.6",
     "object-assign": "^4.0.1",
-    "oust": "^0.2.3",
+    "oust": "git://github.com/akrawchyk/oust",
```

Comment:
So I'm wondering if https://github.com/addyosmani/critical/pull/129 is a better idea. I'm pleased `oust` supports preload links, but I noticed `inline-critical` automates the preload pattern.

This still might be useful, however, if there are manually added preload links in the HTML that `critical` will parse. Perhaps both PR's make sense?